testnets/nobletestnet/assetlist.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "noble",
+  "assets": [
+    {
+      "description": "The controlled staking asset for Noble Chain",
+      "denom_units": [
+        {
+          "denom": "ustake",
+          "exponent": 0,
+          "aliases": [
+            "microstake"
+          ]
+        },
+        {
+          "denom": "stake",
+          "exponent": 6
+        }
+      ],
+      "base": "ustake",
+      "name": "Stake",
+      "display": "stake",
+      "symbol": "STAKE"
+    },
+    {
+      "description": "Love is a test tokenfactory asset controlled by the Strangelove Team",
+      "denom_units": [
+        {
+          "denom": "ulove",
+          "exponent": 0,
+          "aliases": [
+            "microlove"
+          ]
+        },
+        {
+          "denom": "love",
+          "exponent": 6
+        }
+      ],
+      "base": "ulove",
+      "display": "ulove",
```

Comment:
again, I assume the display denom unit is meant to be 'love' (the one with 6 exponent)

Reply:
https://github.com/strangelove-ventures/chain-registry/commit/30d279928aa54121917d9f4969582ef961cb8319